packages/react/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,134 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import {
+  UIComponent,
+  UIComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  customPropTypes,
+  childrenExist,
+  rtlTextContainer,
+} from '../../lib'
+import { RenderResultConfig } from 'src/lib/renderComponent'
+import { defaultBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+import { ComponentEventHandler, ReactProps, ShorthandValue } from '../../types'
+import { Box, Icon } from '../..'
+
+export type AlertSlotClassNames = Record<'content' | 'closeIcon', string>
+
+export interface AlertProps extends UIComponentProps, ContentComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default defaultBehavior
+   */
+  accessibility?: Accessibility
+
+  /** Controls Alert's relation to neighboring items. */
+  attached?: boolean | 'top' | 'bottom'
+
+  /** Icon used for dismissing the Alert, 'close' by default. */
+  closeIcon?: ShorthandValue
+
+  /** An alert may be formatted to display a danger message. */
+  danger?: boolean
+
+  /** An alert may be formatted to display information. */
+  info?: boolean
+
+  /**
+   * The callback used when the user chooses to hide the Alert.
+   * @param {SyntheticEvent} event - React's original SyntheticEvent.
+   * @param {object} data - All props and proposed value.
+   */
+  onClose?: ComponentEventHandler<AlertProps>
+
+  /** An alert may be formatted to display a successful message. */
+  success?: boolean
+
+  /** An alert may be formatted to display a warning message. */
+  warning?: boolean
+}
+
+/**
+ * A Alert displays information that explains nearby content.
+ */
+class Alert extends UIComponent<ReactProps<AlertProps>, {}> {
+  static displayName = 'Alert'
+  static className = 'ui-alert'
+
+  static slotClassNames: AlertSlotClassNames = {
+    content: `${Alert.className}__content`,
+    closeIcon: `${Alert.className}__closeIcon`,
+  }
+
+  static propTypes = {
+    ...commonPropTypes.createCommon(),
+    attached: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['top', 'bottom'])]),
+    closeIcon: customPropTypes.itemShorthand,
+    danger: PropTypes.bool,
+    info: PropTypes.bool,
+    onClose: PropTypes.func,
+    success: PropTypes.bool,
+    warning: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    accessibility: defaultBehavior,
+    closeIcon: 'close',
```

Comment:
I would really try to avoid using default values for icon's name as they are part of the theme, but I don't have better proposal at this moment.. We have the issue in other components as well, I am just expressing my thoughts here..

Reply:
😿 same thoughts exactly but we need this right now

Reply:
removed defaults